### ISSUE TITLE
OCR Text in DocViewer breaks after each word

### ISSUE THREAD

**Issue by eyeseast** [MuckRock/documentcloud-frontend#198]
There was some weird quirk of the implementation of OCR on text pages where each word appears to be wrapped up as it's own paragraph or something. Dylan implemented a workaround so it now copies and pastes correctly, but I tried it in a screen reader and and it looks like a big issue with accessibility.

**Comment by eyeseast** [comment]
Tested this just now, using the built-in text-to-speech on MacOS, and it seems fine. (Actually fine in both current and legacy mode.) Can we reproduce this?

**Comment by eyeseast** [comment]
I tested two versions of the same document using the built-in MacOS VoiceOver utility, both to navigate the page and read the document.

- Current: https://www.documentcloud.org/documents/25929382-parents-under-pressure/
- Legacy: https://legacy.www.documentcloud.org/documents/25929382-parents-under-pressure/

This is a well-formatted government document, something that clearly started as a Word document. Text is embedded and well-formatted. Someone with a screen reader should be able to listen to the text of this document and understand it.

There's a lot to navigate before you get to the page text, so we could make that smoother, but once I got there, it's easy to see the improvements we've made. The legacy version breaks out each word, and because each page is an image, the screen reader announces "image" a lot. The new one sounds like it's just reading the text.

I'm going to call this done.
